core/iwasm/include/wasm_export.h

```diff
@@ -212,6 +212,14 @@ typedef struct wasm_val_t {
 } wasm_val_t;
 #endif
 
+typedef enum {
+    BH_LOG_LEVEL_FATAL = 0,
+    BH_LOG_LEVEL_ERROR = 1,
+    BH_LOG_LEVEL_WARNING = 2,
+    BH_LOG_LEVEL_DEBUG = 3,
+    BH_LOG_LEVEL_VERBOSE = 4
```

Comment:
if these are for wasm_runtime_set_log_level, it seems a bit odd to use BH_ prefix. how do you think?

Reply:
How about changing to `WASM_LOG_LEVEL_*`? Or is there other suggestion?

Reply:
> How about changing to `WASM_LOG_LEVEL_*`?

it sounds better to me.

Reply:
Good to me too. @eloparco, could you help update the code? Thanks.